### PR TITLE
Serve API spec for Metadata service

### PR DESCRIPTION
**Pull request by akgalwas**

### CODE REVIEW

File: docs/application-connector/docs/013-details-api.md [hunk 1]
```diff
@@ -9,3 +9,8 @@ Find the Application Connector API documentation in the included Swagger files.
 - See [this file](assets/metadataapi.yaml) for the Metadata API specification.
 
 For convenient viewing, open these files using [this](https://editor.swagger.io/) Swagger editor.
+
+You can acquire the API specification of Metadata Service for a given version using the following command:
```
Comment (klaudiagrz) [inline docs/application-connector/docs/013-details-api.md]: `the Metadata Service` ?

Comment (akgalwas) [reply docs/application-connector/docs/013-details-api.md]: Done

File: docs/application-connector/docs/013-details-api.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ Find the Application Connector API documentation in the included Swagger files.
...
 For convenient viewing, open these files using [this](https://editor.swagger.io/) Swagger editor.
 
+You can acquire the API specification of the Metadata Service for a given version using the following command:
+```
+curl https://gateway.{CLUSTER_NAME}.kyma.cx/{RE_NAME}/v1/metadataapi.yaml
+```
+
 You can acquire the API specification for a given version directly from the Connector Service:
 ```
 curl https://connector-service.{CLUSTER_NAME}.kyma.cx/v1/api.yaml
```
Comment (janmedrek) [inline docs/application-connector/docs/013-details-api.md]: Like previous one, perhaps it is worth to consider adding minikube subsection for fetching API spec

File: docs/application-connector/docs/013-details-api.md [hunk 3]
```diff
@@ -11,6 +11,11 @@ Find the Application Connector API documentation in the included Swagger files.
 
 For convenient viewing, open these files using [this](https://editor.swagger.io/) Swagger editor.
 
+You can acquire the API specification of the Metadata Service for a given version using the following command:
+```
+curl https://gateway.{CLUSTER_NAME}.kyma.cx/{RE_NAME}/v1/metadataapi.yaml
```
[the remark this answers is not in the document]
Comment (janmedrek) [reply docs/application-connector/docs/013-details-api.md]: The same goes for latter API spec from connector-service

[the remark this answers is not in the document]
Comment (akgalwas) [reply docs/application-connector/docs/013-details-api.md]: Done

**Pull request merged**